nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'method should be implemented in concrete class'

```

Comment:
- [ ] [OPTIONAL] good job adding the error message, you might even want to use string interpolation here to make the message interactive like below:
```
"#{self.class} has not implemented method '#{__method__}'"
```